Dockerfile.arm

```diff
@@ -1,23 +1,31 @@
+# Requires binfm_misc registration for aarch64                                                               
+# https://github.com/multiarch/qemu-user-static#binfmt_misc-register 
 ARG DOTNET_VERSION=3.0
 
 
-FROM microsoft/dotnet:${DOTNET_VERSION}-sdk-stretch-arm32v7 as builder
+FROM multiarch/qemu-user-static:x86_64-arm as qemu                                                       
+FROM alpine as qemu_extract                                                                                  
+COPY --from=qemu /usr/bin qemu-arm-static.tar.gz                                                         
+RUN tar -xzvf qemu-arm-static.tar.gz
+
+FROM microsoft/dotnet:${DOTNET_VERSION}-sdk-stretch as builder
 WORKDIR /repo
 COPY . .
 #TODO Remove or update the sed line when we update dotnet version.
 RUN export DOTNET_CLI_TELEMETRY_OPTOUT=1 \
  && find . -type f -exec sed -i 's/netcoreapp2.1/netcoreapp3.0/g' {} \; \
- && dotnet clean -maxcpucount:1 \
```

Comment:
That `maxcpucount` was there for a reason (it borked the process under certain conditions, like running stuff under `fakeroot`). It might not be strictly needed here, but I think we'd better leave it be to have our stuff as unified as possible.

Reply:
It also shouldn't need cleaning regardless.

Reply:
Unless I'm missing something, that shouldn't be needed as this portion of the build is on x86_64, but would possible slow the build process.  

I probable should have made it clearer earlier that this setup will not build on arm/arm64 but will allow docker hub to automate the creation of docker images that will run on arm/arm64.

If we want to retain the ability to build on arm, I could push theses Dockerfiles as Dockerfile.arm-x86_64 and Dockerfile.arm64-x86_64 or similar for docker hub builds.

Reply:
@EraYaN why not?

Reply:
Yup - output dir `/jellyfin` is created exclusively inside the container.